portfolio/pom.xml

```diff
@@ -17,6 +17,7 @@
   </properties>
 
   <dependencies>
+
```

Comment:
Nit: remove empty line.

Reply:
Well, maybe you can ignore this comment. I think it's a 50-50 call. Fine to leave as is, but I seem to se we usually only use empty lines between sub-compoents (between <denepdency>).